dask/dataframe/tests/test_approximate_size.py

```diff
@@ -0,0 +1,16 @@
+import dask.dataframe as dd
+import dask.async
+import pandas as pd
+import numpy as np
+
+def test_smoke():
```

Comment:
This could be expanded to a few different datasets with different distributions of elements